app/scripts/migrations/133.2.ts

```diff
@@ -0,0 +1,57 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 133.2;
+
+/**
+ * This migration removes tokens on mainnet with the
+ * zero address, since this is not a valid erc20 token.
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly
+ * what we persist to disk.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(state: Record<string, unknown>): void {
+  if (
+    !hasProperty(state, 'TokensController') ||
+    !isObject(state.TokensController) ||
+    !isObject(state.TokensController.allTokens)
+  ) {
+    return;
+  }
+
+  const chainIds = ['0x1'];
+
+  for (const chainId of chainIds) {
+    if (!isObject(state.TokensController.allTokens[chainId])) {
+      continue;
+    }
+
+    for (const [account, tokens] of Object.entries(
+      state.TokensController.allTokens[chainId],
```

Comment:
`tsc` is giving `Object is of type 'unknown'` on these references to `state.TokensController.allTokens[chainId]`, no matter how I guard it with `isObject`.  Not sure how to satisfy it.